SPECS/cloud-init/cloud-init.spec

```diff
@@ -1,15 +1,15 @@
 Summary:        Cloud instance init scripts
 Name:           cloud-init
-Version:        23.3
-Release:        1%{?dist}
+Version:        23.4.1
+Release:        2%{?dist}
 License:        GPLv3
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Base
 URL:            https://launchpad.net/cloud-init
 Source0:        https://launchpad.net/cloud-init/trunk/%{version}/+download/%{name}-%{version}.tar.gz
 Source1:        10-azure-kvp.cfg
-Patch0:         overrideDatasourceDetection.patch
+Patch0:         Retain-exit-code-in-cloud-init-status-for-recoverabl.patch
```

Comment:
nit in patch's name: recoverabl -> recoverable

Reply:
This file name comes from the output of `git format-patch` (minus the initial numbering). Will keep this naming generation consistent